bfvmm/src/vmm/src/vmm_intel_x64.cpp

```diff
@@ -356,34 +357,25 @@ vmm_intel_x64::disable_vmx_operation()
 vmm_error::type
 vmm_intel_x64::create_vmxon_region()
 {
-    // if (m_memory_manager->alloc_page(&m_vmxon_page) != memory_manager_error::success)
-    // {
-    //     std::cout << "create_vmxon_region failed: "
-    //               << "out of memory" << std::endl;
-    //     return vmm_error::out_of_memory;
-    // }
-
-    // if (m_vmxon_page.size() < vmxon_region_size())
-    // {
-    //     std::cout << "create_vmxon_region failed: "
-    //               << "the allocated page is not large enough:" << std::endl
-    //               << "    - page size: " << m_vmxon_page.size() << " "
-    //               << "    - vmxon/vmcs region size: " << vmxon_region_size()
-    //               << std::endl;
-    //     return vmm_error::not_supported;
-    // }
-
-    // if (((uintptr_t)m_vmxon_page.phys_addr() & 0x0000000000000FFF) != 0)
-    // {
-    //     std::cout << "create_vmxon_region failed: "
-    //               << "the allocated page is not page aligned:" << std::endl
-    //               << "    - page phys: " << m_vmxon_page.phys_addr()
-    //               << std::endl;
-    //     return vmm_error::not_supported;
-    // }
+    m_vmxon_page = new char[4096];
```

Comment:
same thing as above. Either use std::array if it will work, or make_unique<char[]>(4096)